component/api/src/main/java/org/exoplatform/social/core/storage/api/SpaceStorage.java

```diff
@@ -762,4 +762,22 @@ default int countPendingSpaceRequestsToManage(String username) {
   default int countExternalMembers(Long spaceId) {
     throw new UnsupportedOperationException();
   }
+
+  /**
+   * Get common spaces between two users
+   *
+   * @param userId        connected user id
+   * @param otherUserId visited profile user id
+   * @return list of common spaces between two users in param
+   */
+  public List<Space> getCommonSpaces(String userId, String otherUserId, int offset, int limit) ;
```

Comment:
```suggestion
  default List<Space> getCommonSpaces(String userId, String otherUserId, int offset, int limit) {
     throw new UnsupportedOperationException();
  }
```

We need to define a default behaviour in case some module implements a mock of this interface, so that it prevent them to fail at build